templates/takeovers/_iot-devops_takeover.html

```diff
@@ -0,0 +1,53 @@
+<section class="p-strip--dark is-shallow p-takeover js-takeover">
+    <div class="row u-clearfix u-vertically-center">
+      <div class="col-7">
+        <h1>Transform the IoT DevOps journey</h1>
+        <p class="p-heading--four u-sv2">
+          Using Ubuntu Core and snaps to accelerate the Linux development cycle
+        </p>
+        <!-- Change link -->
+        <a
+          href="/engage/iot-devops"
+          class="p-button--positive u-hide--small" 
+          onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'Transform the IoT DevOps journey', 'eventLabel' : 'Download the whitepaper', 'eventValue' : undefined });"
```

Comment:
No need for inline onclick events on takeovers, they are already automatically generated.